client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -114,8 +117,8 @@ export class CommentDetailAuthor extends Component {
 	}
 
 	authorMoreInfo() {
-		if ( ! this.showMoreInfo() ) {
-			return;
+		if ( ! this.showMoreInfo() || 'comment' !== this.props.commentType ) {
```

Comment:
The second part of the check looks redundant (the expression doesn't depend on it). It will only be evaluated when `! this.showMoreInfo()` is `false`, and for that to happen `'comment' === this.props.commentType` must be `true`.

Reply:
You're right! I missed this other check